wifite/attack/all.py

```diff
@@ -54,7 +54,7 @@ def attack_single(cls, target, targets_remaining):
             # WPA can have multiple attack vectors:
 
             # WPS
-            if target.wps != False:
+            if target.wps:
```

Comment:
This is awkward... I recently changed `Target.wps` from a `bool` (binary) to `bool + None` (ternary).

* `True`: Target is WPS and **unlocked**
* `False`: Target is **not** WPS.
* `None`: Target is WPS but **locked**

See https://github.com/derv82/wifite2/blob/838ea43a73a6c20f1a625b19eb5347f328eb8bae/wifite/tools/tshark.py#L199-L204

I wanted to allow people to see *Locked* WPS targets, and still attack them if desired.

I should have added another field to `model.target` (such as `target.wps_locked`), but I was lazy & overloaded what `wps` meant.